viz.yaml

```diff
@@ -45,3 +45,48 @@ fetch:
 process:
 visualize:
 publish:
+  -
+    id: headerCSS
```

Comment:
I wasn't getting any of its CSS until I added it to the viz.yaml though...thoughts?